tests/unit/s2n_async_pkey_test.c

```diff
@@ -40,6 +43,7 @@ const uint32_t test_decrypted_size = sizeof(test_decrypted_data);
 uint8_t offload_callback_count = 0;
 
 typedef int (async_handler)(struct s2n_connection *conn);
+static int async_handler_sign_operation_called = 0;
 
```

Comment:
Does this need to be signed? Can we use a bool or a uint8_t or instead?

Reply:
Good point. 

Updated the flag to bool type. 
Done